db/schema.rb

```diff
@@ -26,6 +26,7 @@
 
   create_table "users", force: :cascade do |t|
     t.string "fax_number", null: false
+    t.string "password_digest", null: false
```

Comment:
I don't think this is supposed to be in this PR. Can you revert it by running `git checkout master db/schema.rb`?